### DISCUSSION TITLE
Get solution data after reducing matrix

### DISCUSSION THREAD

**Discussion by neeraj-lohia** [ansys/pyaedt#3988] [Q&A] [answered]
Hi, 
I am working on a script where,
acl1 = q3d.post.available_report_quantities(quantities_category="ACL Matrix")
sol1 = post.get_solution_data(acl1)
then we can get the complete solution data for all nets

but if we want it for a particular reduced matrix, 
acl2 = q3d.post.available_report_quantities(quantities_category="ACL Matrix", **context="some_rm"**)
sol2 = post.get_solution_data(acl2)
then it says "No trace data for quantity: acl2", and returns a **False** value

Is there any other way to get this data?

**Comment by maxcapodi78** [comment, marked as answer]
Hi @neeraj-lohia ,
you should try the following:
```
acl2 = q3d.post.available_report_quantities(quantities_category="ACL Matrix", context="some_rm")
sol2 = post.get_solution_data(acl2, context="some_rm")
```

**Reply by neeraj-lohia** [reply]
yes it works, i already did it. thanks for your time
